src/hardware/mixer.cpp

```diff
@@ -2973,9 +2973,12 @@ static void init_mixer_dosbox_settings(Section_prop& sec_prop)
 
 	auto int_prop = sec_prop.Add_int("rate", OnlyAtStart, DefaultSampleRateHz);
 	assert(int_prop);
-	int_prop->Set_values(
-	        {"8000", "11025", "16000", "22050", "32000", "44100", "48000"});
-	int_prop->Set_help("Mixer sample rate in Hz (%s by default).");
+	int_prop->SetMinMax(8000, 96000);
+	int_prop->Set_help(
+	        "Sample rate of DOSBox's internal audio mixer in Hz (%s by default).\n"
+	        "Valid range is 8000 to 96000 Hz. With the default 'negotiate = on' setting,\n"
+	        "the actual selected sample rate depends on what the host OS and audio driver\n"
+	        "support. Use 'negotiate = off' to force arbitrary mixer sample rates.");
```

Comment:
As we discussed on Discord, I think it would be best to always allow SDL to negotiate the sample rate.  In practice, the setting does not matter on any recent OS.

I just tested disabling PulseAudio on my Linux system running Staging directly through ALSA.  Even then, ALSA has a plugin that resamples any arbitrary sample rate you give it to 48,000hz by default and SDL uses that rather than mixing itself.

I would recommend changing this help text to something like:

```
"Sample rate of DOSBox's internal audio mixer in Hz (%s by default).\n"
"Valid range is 8000 to 96000 Hz\n"
"The vast majority of consumer-grade sound hardware uses a native rate of 48,0000Hz.\n"
"Recommend leaving this as is unless you have good reason.\n"
"The OS is likely to resample anything else to 48,0000Hz anyway.\n"
```

This is the best we can do really.  Detecting native sample rate would be difficult because modern sound systems like PulseAudio allow the user to switch between sound devices at runtime (ex. You can switch between headphones and HDMI out while Staging is running and we're none the wiser.)

Reply:
Thanks for the extra testing and looking into the Linux situation @weirddan455. I agree with you, and I came to similar conclusions. The revised description looks good too, I like it. Will make these changes today.